include/ignition/common/detail/SpecializedPlugin.hh

```diff
@@ -25,7 +25,7 @@
 // This preprocessor token should only be used by the unittest that is
 // responsible for checking that the specialized routines are being used to
 // access specialized plugin interfaces.
-#ifdef IGNITION_UNITTEST_SPECIALIZED_PLUGIN_ACCESS
+#ifdef GZ_UNITTEST_SPECIALIZED_PLUGIN_ACCESS
```

Comment:
These are deprecated, can we leave them as `IGNITION`?

Reply:
https://github.com/gazebosim/gz-common/pull/366/commits/603bbc67fca8c93ea4677296c223b434a3277c96

Reply:
Also fixed in gz-plugin